Jenkinsfile

```diff
@@ -365,6 +365,9 @@ def testGo(prefix, packagesToTest) {
     // Make sure we don't accidentally pull in the testing package.
     sh '! go list -f \'{{ join .Deps "\\n" }}\' github.com/keybase/client/go/keybase | grep testing'
 
+    // Make sure vendored stellar external packages haven't changed
+    sh 'make shavendorstellar'
```

Comment:
It'd be better in the `test_linux` section so it's only run once.

Reply:
I moved it, but feel free to give me a line number if still in the wrong place.